src/file_util.cc

```diff
@@ -78,7 +78,7 @@ class StartupDataCppWriter {
     WriteBinaryContentsAsCArray();
     file_ << "};\n";
     snprintf(buffer, sizeof(buffer), "static const int %s_blob_size = %llu;\n",
-             name_, static_cast<unsigned long long>(data_.size()));
+             name_, static_cast<uint64_t>(data_.size()));
```

Comment:
Why this change? This is not correct.

Reply:
I don't like unsigned long long either. But my compiler complains that "%llu" isn't appropriate for uint64_t.
The correct way to do this, then is to use the PRId64 macro. It's horrible, but it works:
`snprintf(buffer, sizeof(buffer), "static const int %s_blob_size = " PRId64 ";\n",

Reply:
wow. ok. never seen PRId64 before. will fix next time.